#! /bin/sh /usr/share/dpatch/dpatch-run
## ast_key_dir.dpatch by Faidon Liambotis <paravoid@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Put public/private keys in DATADIR/keys (aka /usr/share/asterisk/keys)
## DP: instead of VARLIBDIR/keys. Besided being FHS compliant, that's where
## DP: the Makefile installs them and what src/main/asterisk.c does when
## DP: parsing asterisk.conf. Should be forwarded upstream.

@DPATCH@
diff -urNad asterisk-1.4.9~dfsg~/build_tools/make_defaults_h asterisk-1.4.9~dfsg/build_tools/make_defaults_h
--- asterisk-1.4.9~dfsg~/build_tools/make_defaults_h	2006-04-16 01:53:53.000000000 +0300
+++ asterisk-1.4.9~dfsg/build_tools/make_defaults_h	2007-08-06 04:24:38.000000000 +0300
@@ -14,7 +14,7 @@
 #define AST_DATA_DIR   "${INSTALL_PATH}${ASTDATADIR}"
 #define AST_LOG_DIR    "${INSTALL_PATH}${ASTLOGDIR}"
 #define AST_AGI_DIR    "${INSTALL_PATH}${AGI_DIR}"
-#define AST_KEY_DIR    "${INSTALL_PATH}${ASTVARLIBDIR}/keys"
+#define AST_KEY_DIR    "${INSTALL_PATH}${ASTDATADIR}/keys"
 #define AST_DB         "${INSTALL_PATH}${ASTVARLIBDIR}/astdb"
 #define AST_TMP_DIR    "${INSTALL_PATH}${ASTSPOOLDIR}/tmp"
 
